### DISCUSSION TITLE
user management?

### DISCUSSION THREAD

**Discussion by smokinjo** [socioboard/Socioboard-5.0#367] [Q&A]
Nick started a discussion abotu an admin user and creating one.  He did not seem to get a reply, so I am starting a similar one.

Logging in to socioboard was only possible by having the DB edited and adding a user.   Upon installation, I would expect an admin user to be automatically created and assigning of a password. This person woudl then create teams and users.

Where are the features to add users(start with admin?) and teams?   We need a simple set up: 2 teams, and 2-3 users each one.   Basic set up like this should be doable with socioboard.

When we did add a user, it expired after 7 days.

I heard about amember, but for the basic set up like I described, socioboard should have basic user management.  a third party(paid at that for amember) could be used for advanced set ups.   An open source solutions that requires a paid plug in to work is an  interesting open source project.   Or, is there a basic version that can be used for simple set ups?

What is the future of user management in the road map of socioboard?

How can I get my basic teams up and running with out amember?

Thanks

**Comment by sureshbabu1995** [comment]
Hello @smokinjo 

Thanks for the note, We do maintain all these details in DB, if you want to detach the amember (plugin) yes, it can be with changing the code and keeping the constants, and if you set the highest values available at once, then you no need to think about the middleware of amember, you'll get all accesses with the constants

Yes, in the upcoming one, we'll launch the default case with all options (no plugin setup requires)

**Reply by smokinjo** [reply]
Hello @sureshbabu1995

Thanks for this message.  

Sounds like using socioboard with out amember can be done, but needs a hand from a programmer to set it up.

You mentioned a new version.   When do you plan on releasing it?  Will it be version 6?  Or, something else?

Thanks!
